pastas/modelcompare.py

```diff
@@ -854,31 +854,6 @@ def plot_table_diagnostics(
         cols = diags.columns.to_list()[-1:] + diags.columns.to_list()[:-1]
         return self.plot_table(axn=axn, df=diags[cols])
 
-    def share_xaxes(self, axes: List[Axes]) -> None:
```

Comment:
I'd vote for leaving this method but using the new function share_xaxes().

Reply:
Agreed.